src/utils/AttachVariableChangeListener.js

```diff
@@ -18,6 +23,13 @@ const attachVariableChangeListener = title => {
   });
 };
 
+
+/**
+ * Change variables with same tag if one of the variable changes among them i.e. make all variables with same tag have similar value
```

Comment:
Well written! :) Only grammatical fix.

`Updates value of all the instance of variables with the same tags when one of them is manually modified.`